code/src/components/Movies.js

```diff
@@ -0,0 +1,38 @@
+import React, { useState } from 'react';
+import { Link } from 'react-router-dom';
+import '../css/Movies.css';
+
+// This component displays a single movie
+const Movies = ({ title, releaseDate, movieId, poster }) => {
+  // 'isHover' tracks if the mouse is over the movie, 'setIsHover' updates the state
+  const [isHover, setIsHover] = useState(false)
+
+  // This function is called when the mouse enters the movie area
+  const handleMouseEnter = () => {
+    setIsHover(true);
+  };
+  // This function is called when the mouse leaves the movie area
+  const handleMouseLeave = () => {
+    setIsHover(false);
+  };
+
+  // This part shows the movie poster and title on the screen
+  return (
```

Comment:
I like the fact that you've made each movie in the movielist into its own component.